src/main/java/seedu/address/logic/Logic.java

```diff
@@ -80,15 +78,8 @@
     void setGuiSettings(GuiSettings guiSettings);
 
     /**
-     * Returns a List of Strings describing the commands that can be undone.
+     * Get statistics on EntityLists.
      */
-    List<String> getUndoCommandHistory();
-
-    /**
-     * Returns a List of Strings describing the commands that can be redone.
-     */
-    List<String> getRedoCommandHistory();
-
```

Comment:
Context for this removal?

Reply:
it's no longer used. It was for debugging, really. Cos I pritned the string in the command box